lib/plugins/aws/deploy/compile/events/apiGateway/lib/cors.test.js

```diff
@@ -55,16 +55,19 @@ describe('#compileCors()', () => {
         origins: ['*'],
         headers: ['*'],
         methods: ['OPTIONS', 'PUT'],
+        allowCredentials: false,
```

Comment:
Could we remove this setup so that we test that it defaults to `false`?

Then we have one test w/o any `allowCredentials` setup, one which tests if it's set to `true` and one which tests if it's set to `false`.